migrations/versions/f2f57cb2a5b0_add_index_on_access_token.py

```diff
@@ -0,0 +1,34 @@
+"""Add index on access token
+
+Revision ID: f2f57cb2a5b0
+Revises: cb232e6e9425
+Create Date: 2025-02-24 21:33:31.076672
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "f2f57cb2a5b0"
+down_revision: Union[str, None] = "fc5e2283de94"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("oauth2token_access_token_key1", "oauth2token", type_="unique")
```

Comment:
This key constraint name looks to actually be `oauth2token_access_token_key`, at least in the `qa` deployment. The upgrade is currently failing due to `oauth2token_access_token_key1` not being defined on the table.